source/WB-CAN-Firmware.c

```diff
@@ -75,20 +80,39 @@ const uint32_t g_Adc16_12bitFullRange = 4096U;
 #define ADC16_IRQ_HANDLER_FUNC ADC0_IRQHandler
 
 
-#define valvePin BOARD_INITPINS_HS_SWITCH_B_IN0_GPIO
-#define valvePinMask BOARD_INITPINS_HS_SWITCH_B_IN0_GPIO_PIN_MASK
+#define VALVE_PIN BOARD_INITPINS_HS_SWITCH_B_IN0_GPIO
+#define VALVE_PIN_MASK BOARD_INITPINS_HS_SWITCH_B_IN0_GPIO_PIN_MASK
 
 void PWM(uint32_t, uint32_t);
 uint32_t duty_cycle = 10;
 
+uint8_t pwm_active = 0;
+
+
+enum testConfigEnum
+    {
+        POTENTIOMETER, KULITE
+    };
+typedef enum testConfigEnum testConfigType;
```

Comment:
You can shorten this:

```suggestion
typedef enum testConfigEnum
    {
        POTENTIOMETER, 
        KULITE
    } testConfigType;
```

We don't have a style guide or anything (maybe we should make one? But it's probably too far into the project for that to have a worthwhile payoff. I digress) but generally we use `_t` at the end of anything typedef'd to specify that it's a 'custom' type (kind of like `uint8_t` or `int64_t`). So I would change this type of `testConfigType` to something like `testConfig_t`, here and through the rest of the code.